validation/rollup-config_test.go

```diff
@@ -14,3 +14,12 @@ func testRollupConfig(t *testing.T, chain *ChainConfig) {
 	assertIntInBounds(t, "Block Time", chain.BlockTime, standard.BlockTime)
 	assertIntInBounds(t, "Sequencer Window Size", chain.SequencerWindowSize, standard.SequencerWindowSize)
 }
+
+func testOptimismConfig(t *testing.T, chain *ChainConfig) {
```

Comment:
Maybe a comment explaining the history around this? These values used to be hardcoded and not read from the registry. Since we added them to the registry and remove the hardcodes from op-geth, this test protects us from chains which ran an old version of `add-chain`.